src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,22 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    private String addDetailToDetailsString(String details, String detail) {
```

Comment:
No need to create a new method for this. Also the method name is quite confusing. :)